src/main/java/gregtech/common/ConfigHolder.java

```diff
@@ -10,6 +10,14 @@
     @Config.Name("Unofficial Options")
     public static UnofficialOptions U = new UnofficialOptions();
 
+    @Config.Comment("Whether to enable that Steam Multiblocks use Steel instead of Bronze. Default: false")
+    @Config.RequiresMcRestart
+    public static boolean steelMultiblocks = false;
+
+    @Config.Comment("Steam to EU multiplier for Steam Multiblocks. 1.0 means 1 Steam -> 1 EU. 0.5 means 2 Steam -> 1 EU")
+    @Config.RequiresMcRestart
+    public static double steamtoEU = 0.5;
```

Comment:
Please change this config name to "multiblockSteamToEU". It's more clear on its purpose.